lib/KeyboardAwareMixin.js

```diff
@@ -48,6 +48,13 @@ const KeyboardAwareMixin = {
     this.keyboardWillHideEvent.remove()
   },
 
+  scrollToPosition: function(x, y) {
```

Comment:
I would like to have three things before merging this PR:

1. Please insert an space between `function` and the parens.
2. Add Flow annotations.
3. Add also a third parameter to programatically set the `animated` value.

In the end, it would look like this:

```js
scrollToPosition: function (x: number, y: number, animated: bool = false) {
...
}
```